src/modules/pos/validator/components/ClaimRewardsForm/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useRewardsClaimable } from '@pos/reward/hooks/queries';
+import classNames from 'classnames';
+import BoxHeader from '@theme/box/header';
+import { useTranslation } from 'react-i18next';
+import { MODULE_COMMANDS_NAME_MAP } from '@transaction/configuration/moduleCommand';
+import TxComposer from '@transaction/components/TxComposer';
+import { useCurrentAccount } from '@account/hooks';
+import { QueryTable } from '@theme/QueryTable';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import styles from './ClaimRewardsForm.css';
+
+const rewardsClaimableHeader = (t) => [
+  {
+    title: t && t('Token'),
+    classList: `${grid['col-xs-3']}`,
+  },
+  {
+    title: t && t('Reward amount'),
+    classList: `${grid['col-xs-4']}`,
+  },
+  {
+    title: t && t('Fiat'),
+    classList: `${grid['col-xs-1']}`,
+  },
+];
+
+const RewardsClaimableRow = ({ data }) => {
+  const { tokenName, logo, reward, symbol, denomUnits, displayDenom } = data;
+  const amountInFiat = reward;
+  const denom = denomUnits?.find((denomUnit) => denomUnit.denom === displayDenom);
+
+  return (
+    <div className={classNames(styles.rewardsClaimableRow)}>
+      <div className={classNames(styles.logoContainer, rewardsClaimableHeader()[0].classList)}>
+        <img className={styles.logo} src={logo?.png} />
+        <span className={styles.tokenName}>{tokenName}</span>
+      </div>
+      <div className={classNames(rewardsClaimableHeader()[1].classList)}>
+        {`${parseInt(reward, denom)} ${symbol}`}
+      </div>
+      <div className={classNames(rewardsClaimableHeader()[2].classList)}>{amountInFiat}</div>
+    </div>
+  );
+};
+
+const ClaimRewardsForm = ({ nextStep }) => {
+  const { t } = useTranslation();
+  const [currentAccount] = useCurrentAccount();
+  const address = currentAccount?.metadata?.address;
```

Comment:
This can be done instead since the `currentAccount` variable isn't going to be used anywhere else and also currentAccount would also exist since the route to the `ClaimRewardsForm` is a private one

```suggestion
  const [{ metadata: { address } }] = useCurrentAccount();
```

Reply:
Yep, its cleaner like this if no need for optional chaining here. fixed